libraries/botbuilder-core/tests/testAdapter.test.js

```diff
@@ -166,7 +166,7 @@ describe(`TestAdapter`, function () {
         await assert.rejects(
             async () => await adapter.send('test').assertReply('received', 'received failed', 500).startTest(),
             (err) => {
-                assert.match(err.message, /.*Timed out after.*/);
```

Comment:
Up to you if you want to use this, but it looks like `assert.rejects` accepts a `RegExp` directly as the second parameter.

Reply:
Oh, heck yeah I'm going to use that!